### PR TITLE
Changed description of Solar Panels CBM and made them activable

### PR DESCRIPTION
**Pull request by Night-Pryanik**
An alternative to #23317.

Now the panels are activable and cost 10 energy to retract and fold. I also planned to add a possibility to break them in case of a blow to torso, but @illi-kun suggested to wait with this till 0.D.

### CODE REVIEW

File: data/json/bionics.json [hunk 1]
```diff
@@ -859,9 +859,10 @@
     "id": "bio_solar",
     "type": "bionic",
     "name": "Solar Panels",
-    "description": "You have a few solar panels installed.  While in direct sunlight, your power level will slowly recharge.",
+    "description": "You have a small solar panels contained in a box on your back.  On activation they will retract in a butterfly wings manner, and while in direct sunlight, your power level will slowly recharge.",
```
Comment (illi-kun) [inline data/json/bionics.json]: "a \\...\ panels" is incorrect.

File: data/json/bionics.json [hunk 2]
```diff
@@ -859,9 +859,10 @@
     "id": "bio_solar",
     "type": "bionic",
     "name": "Solar Panels",
-    "description": "You have a few solar panels installed.  While in direct sunlight, your power level will slowly recharge.",
+    "description": "You have small solar panels contained in a box on your back.  On activation they will retract in a butterfly wings manner, and while in direct sunlight, your power level will slowly recharge.",
     "occupied_bodyparts": [ [ "TORSO", 20 ] ],
-    "flags": [ "BIONIC_POWER_SOURCE" ]
+    "flags": [ "BIONIC_POWER_SOURCE", "BIONIC_TOGGLED" ],
+    "act_cost": 10
```
Comment (illi-kun) [inline data/json/bionics.json]: How this balanced against to produced amount of power? Did you test this? Do you have any assumption how this should be balanced ("activation energy will be restored by 1hr on sun" or something in this way?)
Why there is no ```deact_cost```?

Comment (Night-Pryanik) [reply data/json/bionics.json]: I wasn't even aware of deactivation cost. Added.

I tested only in bright lighting. The produced power is 25, so it's 2,5 times the activation cost. Honestly I think that's too much for a one-time power add.

**Comment by Zireael07** [comment]
Much much better alternative - and the CBM has a downside now instead of being "yaay free power"

**Comment by smolbird** [comment]
Infinitely better than removing them or moving them to a mod.

Gameplay > realism

**Comment by kevingranade** [comment]
No change, mod-only or remove are the options.

**Comment by John-Candlebury** [comment]
>No change, mod-only or remove are the options.

Why, this makes sense. Its genuinely believable when it comes to how it works, fold able solar panels are a thing. The only aspect missing for it to work is increased solar panel efficiency, and that's actively improving in the real world too. Never mind its interesting from a game play angle

I thought cataclysm realism was supposed to take the backseat when it came to bionics and the like. The whole: "Modern day with future elements" shtick".

**Comment by illi-kun** [comment]
Jenkins rebuild

**Comment by kevingranade** [comment]
Merged #23317.

**Pull request closed without merging**